pkg/server/service.go

```diff
@@ -16,14 +15,20 @@ var (
 	ErrConfigNotDeleted  = errors.New("config not deleted")
 )
 
-func insertOneConfig(ctx context.Context, cfg webhooks.ConfigUser, store storage.Store) (string, error) {
+func insertOneConfig(ctx context.Context, cfg webhooks.ConfigUser, store storage.Store) (webhooks.Config, error) {
 	id, err := store.InsertOneConfig(ctx, cfg)
 	if err != nil {
-		return "", errors.Wrap(err, "store.insertOneConfig")
+		return webhooks.Config{}, errors.Wrap(err, "store.insertOneConfig")
 	}
 
 	sharedlogging.GetLogger(ctx).Debug("insertOneConfig: id: ", id)
-	return id, nil
+	if cur, err := store.FindManyConfigs(ctx, map[string]any{webhooks.KeyID: id}); err != nil {
```

Comment:
Why this additional check?

Reply:
Oh, ok, you fetch the updated response from database. Unless you're relying on generated field database side, this extra call seems to be useless.